### DISCUSSION TITLE
A cheatsheet

### DISCUSSION THREAD

**Discussion by Forairaaaaa** [lovyan03/LovyanGFX#344] [General]
**Carefully written requests are more likely to be given priority.**
**丁寧に記述された要望は優先して対応される可能性が高くなります。**

Hi, I found this "cheatsheet" in raylib, which is really helpful, I hope we can have one in LovyanGFX  ;)
https://www.raylib.com/cheatsheet/cheatsheet.html
Thanks~

**Comment by tobozo** [comment]
Hi, thanks for your feedback :+1: 

A cheatsheet would be neat, but the project changes often so this means a new cheatsheet must be produced every time a new release is published.

We tried to document functions with [doxygen](https://lovyangfx.readthedocs.io/en/master/05_contributions.html) in the past but the translation tasks became a tech debt that was never paid, plus some extra time had to be spent during development to comply to doxygen's requirements.

The current output from doxygen is too dense to qualify as cheatsheet content anyway.

![image](https://user-images.githubusercontent.com/1893754/215464739-44bfbcfd-a33c-46c8-b521-b37d96105d5a.png)

Any suggestion (tool, workflow, methodology) to fully automate this without creating additional tech debt is welcome.

**Reply by Forairaaaaa** [reply]
I don't have much good idea for this :( , but thanks a lot for all the working~
